packages/grafana-ui/src/components/Table/FilterList.tsx

```diff
@@ -24,7 +24,7 @@ export const FilterList: FC<Props> = ({ options, values, onChange }) => {
     searchFilter,
   ]);
   const gutter = parseInt(theme.spacing.sm, 10);
-  const height = useMemo(() => Math.min(items.length * ITEM_HEIGHT, MIN_HEIGHT) + gutter, [items]);
+  const height = useMemo(() => Math.min(items.length * ITEM_HEIGHT, MIN_HEIGHT) + gutter, [gutter, items.length]);
```

Comment:
Interesting this one, gutter will always be a constant value but I can see why eslint complains.